jsm.go

```diff
@@ -24,41 +24,41 @@ import (
 
 // JetStreamManager is the public interface for managing JetStream streams & consumers.
 type JetStreamManager interface {
```

Comment:
fyi - Java also has an accessor to get stream names.

Reply:
That is a good point, that API is not exposed in the `JetStreamManager` interface actually. Maybe we should add it too?

```go
GetStreamBySubject(subj string) (*StreamInfo, error)
```

https://github.com/nats-io/nats.go/blob/master/js.go#L655-L662